beancount_n26/__init__.py

```diff
@@ -235,40 +245,62 @@ def extract(self, file_, existing_entries=None):
             for index, line in enumerate(reader):
                 meta = data.new_metadata(file_.name, index)
 
-                s_amount_eur = self._translate('amount_eur')
-                s_amount_foreign_currency = self._translate(
-                    'amount_foreign_currency'
-                )
-                s_payee = self._translate('payee')
-                s_payment_reference = self._translate('payment_reference')
-                s_type_foreign_currency = self._translate(
-                    'type_foreign_currency'
-                )
+                postings = []
 
-                if line[s_amount_eur]:
-                    amount = Decimal(line[s_amount_eur])
-                    currency = 'EUR'
-                else:
-                    amount = Decimal(line[s_amount_foreign_currency])
+                if line[s_amount_foreign_currency]:
```

Comment:
No, because the exchange rate does not include transferwise fees. If 1 EUR = 1 CHF, the exchange_rate would be 1.0, but we would still have xEUR of fees

Checking if fees == 0 should be the best way to do that in my opinion 

Proposed patch in 8e78590c999b48f4366b94a144cfa6caa447e0c1

Reply:
Good point. Then let's use the `fees == 0` check for this.